docs/nodes/qgb-orchestrator.md

```diff
@@ -17,6 +17,10 @@ The orchestrator does the following:
 
 The orchestrator connects to a separate P2P network than the consensus or the data availability one. So, we will provide bootstrappers for that one.
 
+Bootstrapper for the Blockspace Race is:
+
+* `/dns/bootstr-incent-1.celestia.tools/tcp/30000/p2p/12D3KooWSGZ2LXW2soQFHgU82uLfN7pNW5gMMkTnu1fhMXG43TvP`
+
```

Comment:
```suggestion

Make sure to specify it using the `-b` flag when running the orchestrator.

```

Reply:
Also, I would put this after the next paragraph and the logs, because in here, the paragraphs surrounding it are a bit connected